### ISSUE TITLE
Unable to use GPS location without (temporarily) enabling a "Network-based Geolocation module".

### ISSUE THREAD

**Issue by jaimet** [Divested-Mobile/Maps#5]
Hi.

I would like to use GMaps WV with my location provided by my GPS. I have given "Location Access" to GMaps WV, but when I start GMaps WV and click on the "My Location" button in GMaps WV, the spinner (inside the button) just spins forever and the app never finds my location.

Interestingly, I can [work around](https://forum.xda-developers.com/t/lineageos-for-microg.3700997/post-88349403) this problem by installing [Déjà Vu](https://github.com/n76/DejaVu) and enabling it just long enough for GMaps WV to obtain a GPS fix (which occurs very quickly) - I can then disable Déjà Vu, and GMaps WV continues to update my GPS location despite Déjà Vu (now) being disabled. If I close GMaps WV and re-open it, again it will fail to find my GPS location until I "jump start" it with Déjà Vu.

I think this is a bug because I ***presume*** that I do not need to have a "Network-based Geolocation module" enabled in order to get my location via GPS. I might be wrong.

Edit (to add device details):
Device: OnePlus 5T
Rom: [LineageOS for microG v18.1 20220516]( https://download.lineage.microg.org/dumpling/lineage-18.1-20220516-microG-dumpling.zip)
GMaps WV v1.8

**Comment by SkewedZeppelin** [comment]
I tested the location function on my DivestOS which has no such network location providers at all working fine.

You're also missing two plus years of security fixes on that outdated OS.

**Comment by jaimet** [comment]
> I tested the location function on my DivestOS which has no such network location providers at all working fine.

Does this mean that only bugs on DivestOS are valid? If yes, then I will close this bug report. (FWIW, [I am not the only person with this issue](https://forum.xda-developers.com/t/lineageos-for-microg.3700997/post-88349403) but I presume that kurtn is also running "LineageOS for microG".)

**Comment by SkewedZeppelin** [comment]
> Does this mean that only bugs on DivestOS are valid?

No, I'm just saying there may be something funky when UnifiedNlp is a fused provider with no backends enabled.

**Comment by ancheno** [comment]
Same problem here with motorola edge 30, android 13 build T1RDS33.116-33-3-2, gmpas from fdroid rev 1.8 of 31 may 2023
